### PR TITLE
VizTooltip: Wrap labels to new lines when > ~50% screen width

### PR DESCRIPTION
**Pull request by leeoniya**
Fixes https://github.com/grafana/support-escalations/issues/11620

### CODE REVIEW

File: packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ enum LabelValueTypes {
 
 const SUCCESSFULLY_COPIED_TEXT = 'Copied to clipboard';
 const SHOW_SUCCESS_DURATION = 2 * 1000;
+const HZ_PX_PER_CHAR = 7;
```
Comment (nmarrs) [inline packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx:30]: what does `HZ` mean in this context? horizontal? might be worth spelling out here 😅

**Review by nmarrs — APPROVED** [review]
minor nit but LGTM

File: packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx (hunk 1 above)
Comment (leeoniya) [reply packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx:30]: `APPROXIMATE_HORIZONTAL_PIXELS_PER_CHARACTER`

Comment (nmarrs) [reply packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx:30]: haha `HORIZONTAL_PX_PER_CHAR` is fine by me, I just think the `HZ` is a little confusing but the other abbreviations are pretty standard 😬

**Review by adela-almasan — APPROVED** [review]

**Pull request merged**